SPECS/dhcpcd/dhcpcd.spec

```diff
@@ -0,0 +1,83 @@
+%global debug_package   %{nil}
```

Comment:
Do we really need to disable the debuginfo build for this package? Was there an error that caused the need for this line?

Reply:
Initial added this for a build error. After rebuilding looks like we don't need it and removed